.devcontainer/Dockerfile.llvm-project

```
@@ -0,0 +1,20 @@
+# By default, use ubuntu:focal
+ARG BASE_IMAGE="onnxmlirczar/onnx-mlir-llvm-static:amd64"
+FROM ${BASE_IMAGE}
+
+RUN apt-get update
+
+# 2) Instal optional packages, comment/uncomment/add as you see fit.
+RUN apt-get install -y vim bash valgrind clang-format lsb-release wget software-properties-common ssh-client gdb
+RUN python3 -m pip install -U pip
```

```suggestion
RUN python3 -m pip install -U pip && pip install commitizen
```
You can just add the pip install for commitizen here.